src/main/java/fi/dy/masa/tweakeroo/util/InventoryUtils.java

```diff
@@ -581,6 +599,12 @@ else if (getBaseBlockBreakingSpeed(testedStack, state) == getBaseBlockBreakingSp
         return false;
     }
 
+    private static boolean toolHasSilkTouch(ItemStack stack)
+    {
+        // 1.21 way of checking for Silk Touch according to the Fabric upgrade blog post.
```

Comment:
https://fabricmc.net/2024/05/31/121.html#Enchantments:~:text=EnchantmentHelper%23hasAnyEnchantmentsIn(stack%2C%20EnchantmentTags%23PREVENTS_BEE_SPAWNS_WHEN_MINING)

Reply:
There are better ways to do this, such as using 'getEnchantmentLevel()` from my code.

Reply:
That's what I started with and implemented for 1.20.6... Then I realised I was on the 1.20.6 branch and switched to 1.21 :D Apparently checking for a specific enchantment is not as clean anymore, they're trying to get away from enchantment names over to their effects without actually listing all the possible effects -.-'